assets/js/formvalidate.js

```diff
@@ -0,0 +1,19 @@
+const form = document.getElementById('#contact_form');
+
+form.addEventListener('submit', (e) => {
+    const email = document.getElementById('email').value;
+    const errorMsgElement = document.querySelector('.error_msg');
+    const errorMsg = 'Email should be in lower case.';
+    e.preventDefault();
+    const regex = /[A-Z]/;
+    if (!regex.test(email.value)) {
+      form.submit();
```

Comment:
Consider replacing the whole of this section with the code below.

    const regex = /[A-Z]/;
    if (regex.test(email.value)) {
	e.preventDefault();